diff --git a/node_modules/inquirer-press-to-continue/prompt.d.ts b/node_modules/inquirer-press-to-continue/prompt.d.ts
index d1f961f..34e8831 100644
--- a/node_modules/inquirer-press-to-continue/prompt.d.ts
+++ b/node_modules/inquirer-press-to-continue/prompt.d.ts
@@ -15,6 +15,7 @@ declare module 'inquirer' {
      * The type of the answers.
      */
     interface PressToContinuePrompt<T extends inquirer.Answers = inquirer.Answers> extends PressToContinuePromptOptions<T> {
+        name: string;
         /**
          * @inheritdoc
          */
